sklearn/tests/test_build.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pytest
+import textwrap
+
+from sklearn import __version__
+from sklearn.utils._openmp_helpers import _openmp_supported
+
+
+def test_openmp_supported():
+    # Check that sklearn is built with OpenMP supported.
+    # This test can be disabled by setting the environment variable
+    # ``SKLEARN_SKIP_OPENMP_TEST``.
+    if os.getenv("SKLEARN_SKIP_OPENMP_TEST"):
+        pytest.skip("test explicitly disabled (SKLEARN_SKIP_OPENMP_TEST)")
```

Comment:
So concretely we have a whole new CI job only for these 2 lines.
Kinda sad but OK.

Reply:
not really because we want to test that all functions pass the test when building without openmp (hgbt, sparse_manhattan, t-sne, ...)